crates/ra_ide_api/src/inlay_hints.rs

```diff
@@ -0,0 +1,189 @@
+use crate::{db::RootDatabase, FileId};
+use hir::{HirDisplay, Ty};
+use ra_syntax::ast::Pat;
+use ra_syntax::{
+    algo::visit::{visitor, Visitor},
+    ast::{self, PatKind, TypeAscriptionOwner},
+    AstNode, SmolStr, SourceFile, SyntaxNode, TextRange,
+};
+
+#[derive(Debug, PartialEq, Eq)]
+pub enum InlayKind {
+    LetBinding,
+    ClosureParameter,
+}
+
+#[derive(Debug)]
+pub struct InlayHint {
+    pub range: TextRange,
+    pub text: SmolStr,
+    pub inlay_kind: InlayKind,
```

Comment:
The `inlay_` prefix doesn't add any information here and can be safely dropeed: `kind: InlayKind`

Reply:
Fixed.